### PR TITLE
Minor style updates

### PR DESCRIPTION
**Pull request by isunaslabs**
-Updated button cursor style

### CODE REVIEW

**Comment by isunaslabs** [comment]
@JayJay1024 Hey Jay. Please help me reviewing this code. Thanks for your time.

File: packages/darwinia-ui/src/components/Button/styles.scss [hunk 1]
```diff
@@ -44,4 +44,5 @@
 
 .dw-btn-spinner-custom-mask {
   background-color: rgb(0 0 0 / 40%) !important;
+  cursor: not-allowed;
```
Comment (JayJay1024) [inline packages/darwinia-ui/src/components/Button/styles.scss:47]: Will this style affect the loading (large loading) here? The large loading does not need to `cursor: not-allowed`

image

Comment (isunaslabs) [reply packages/darwinia-ui/src/components/Button/styles.scss:47]: No it won't. This behavior is only added on the Button component's spinner as you can see from the commit on file path packages/darwinia-ui/src/components/Button/styles.scss

Comment (isunaslabs) [reply packages/darwinia-ui/src/components/Button/styles.scss:47]: Please let me know if further clarification is needed. Thanks

**Review by JayJay1024 — APPROVED** [review]

**Pull request merged**